### DISCUSSION TITLE
Web Interference

### DISCUSSION THREAD

**Discussion by miltos04** [sherlock-project/sherlock#1022] [Ideas]
Hello to everyone,
Me and my colleague Dimitris are interested in creating a website in JavaScript in order to make the use of the software easier. In the context of our class software engineering in practise we must contribute and collaborate in an open source project. We are students in Athens University of Economics and Business and we are happy to collaborate with anyone who is interested in our idea. 
Please feel free to contact us at any time.
Dimitris account: Dimitrisvoug.

**Comment by sagarspatil** [comment]
@miltos04 We built https://userhunt.co, a web UI for sherlock. Let me know what you think and if you want to contribute.

**Reply by miltos04** [reply]
Hello, yes we still want to contribute. It seems very nice, from the first look I think we could add a dark theme, allow to the user to search multiple accounts at once and give him the option to download a txt file with the results of the search.

**Reply by SauravMaheshkar** [reply]
> @miltos04 We built https://userhunt.co, a web UI for sherlock. Let me know what you think and if you want to contribute.

Noticed the site was down, wanted to link my app as well. https://huggingface.co/spaces/SauravMaheshkar/sherlock
